### PR TITLE
Wire up Papertrail

### LINKED ISSUE #3104: Add support for Papertrail based logging
- [x] Create a Papertrail account with multiple environment specific log groups
- [x] Route as many logs as possible to `syslog`
- [x] Any logs that cannot be routed, use a log file monitoring agent to capture logs and route them to Papertrail
- [x] Attempt to use existing Ansible Papertrail role

### PR DESCRIPTION
**Pull request by rbreslow**
## Overview

This PR adds support for Papertrail based logging (minus modifications to CloudFormation).

Resolves #3104 

## Testing Instructions

- Apply the following diff:

```diff
diff --git a/deployment/ansible/roles/model-my-watershed.base/meta/main.yml b/deployment/ansible/roles/model-my-watershed.base/meta/main.yml
index 8e51f9fa..8295f81a 100644
--- a/deployment/ansible/roles/model-my-watershed.base/meta/main.yml
+++ b/deployment/ansible/roles/model-my-watershed.base/meta/main.yml
@@ -4,4 +4,4 @@ dependencies:
   - { role: "azavea.git" }
   - { role: "azavea.daemontools" }
   - { role: "azavea.postgresql-support" }
-  - { role: "azavea.papertrail", when: "['development'] | is_not_in(group_names)" }
+  - { role: "azavea.papertrail" }
diff --git a/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml b/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml
index 0247f3d6..f8ac36d9 100644
--- a/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml
+++ b/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml
@@ -31,4 +31,3 @@
     - Restart Celery

 - import_tasks: papertrail.yml
-  when: "['development'] | is_not_in(group_names)"
```
- Run `./scripts/vagrant-up.sh`
- See: https://papertrailapp.com/groups/14907082/events

### CODE REVIEW

**Comment by hectcastro** [comment]
A heads up to hold off on merging this until https://github.com/WikiWatershed/model-my-watershed/pull/3109 is merged. After it is merged, we may need to rebase this PR one more time.

File: deployment/ansible/roles/model-my-watershed.base/tasks/papertrail.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- name: Configure remote_syslog
+  template: src="log_files.yml.j2"
+            dest="{{ papertrail_conf }}"
+  notify:
+    - Restart remote_syslog
+
+- name: Add Papertrail user to service group
+  user: name=papertrail
+        append=yes
+        groups=adm
```
[the remark this answers is not in the document]
Comment (rbreslow) [reply deployment/ansible/roles/model-my-watershed.base/tasks/papertrail.yml]: I tried removing the `papertrail` user from `adm` on the `worker` VM:

```bash
sudo deluser papertrail adm
```

And it seems that Papertrail is still able to pick up logs from `/var/log/syslog` and `/var/log/celery/*.log`.

I don't really understand how this is working.

For example, `/var/log/syslog` is not publicly readable:

```bash
-rw-r-----  1 syslog adm    1514393 Jun 27 14:22 syslog
```

And I verified that `papertrail` is no longer in `adm`:

```bash
$ sudo groups papertrail
papertrail : papertrail
```

I'm probably missing something obvious.

**Comment by rbreslow** [comment]
Succeeded by #3103

File: deployment/ansible/roles/model-my-watershed.base/tasks/papertrail.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hectcastro) [reply deployment/ansible/roles/model-my-watershed.base/tasks/papertrail.yml]: Did you restart the service after the group removal? Either way, I think it'll get tested as part of the review for the next PR.

[the remark this answers is not in the document]
Comment (rbreslow) [reply deployment/ansible/roles/model-my-watershed.base/tasks/papertrail.yml]: I restarted the VM.

**Pull request closed without merging**